workflow/rocoto/tasks.py

```diff
@@ -206,9 +214,15 @@ def get_resource(self, task_name):
             # Memory is not required
             memory = task_config.get(f'memory_{task_name}', None)
 
-        if scheduler in ['pbspro']:
-            if task_config.get('prepost', False):
-                memory += ':prepost=true'
+       #The PW_CSP is a AWS (CSPs parameter), if it is on CSPs, cannot define 'memory' here,
+       #Or the arch and cleanup will hang.
+        if ( pw_csp in ['aws', 'azure', 'google'] ):
+            memory = None
+        else:
+            memory = task_config.get(f'memory_{task_name}', None)
+            if scheduler in ['pbspro']:
+                if task_config.get('prepost', False):
+                    memory += ':prepost=true'
```

Comment:
This should not be necessary is `config.resources.AWSPW` unsets the memory variable similar to `Gaea` that @DavidBurrows-NCO added.

Reply:
Following Gaea example, added file config.resorces.AWSPW.